### PR TITLE
Update pipeline to run each version of ansible on each collection

### PR DESCRIPTION
**Pull request by carchi8py**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,43 +1,36 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
 
+      - name: Install ansible (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
+      - name: Run sanity tests (${{ matrix.collection }})
+        run: ansible-test sanity --docker -v --color
+        working-directory: ./ansible_collections/netapp/${{ matrix.collection }}
```
Comment (lonico) [inline .github/workflows/main.yml:36]: is there something preventing us to run the unit tests?
I'm OK if we add them next.

**Review by lonico — APPROVED** [review]

**Comment by lonico** [comment]
fixes https://github.com/ansible-collections/netapp/issues/98

**Review by tadeboro — APPROVED** [review]

**Comment by tadeboro** [comment]
Once changes from this PR land in the main branch, the `netapp.cloudmanager` collection should be ready for inclusion in Ansible 4.

**Comment by carchi8py** [comment]
@tadeboro the change has just been merged

**Pull request merged**